packages/runtime/test/transport/messages.test.ts

```diff
@@ -165,6 +165,14 @@ describe("Message errors", () => {
         expect(result).toBeAnError("Mail.to :: Value is not defined", "error.runtime.requestDeserialization");
     });
 
+    test("should throw correct error for false content type", async () => {
+        const result = await client1.transport.messages.sendMessage({
+            recipients: [client2.address],
+            content: {}
```

Comment:
why is this possible without tricking the compiler?

Reply:
We decided against fixing the type because the schema validator would fuck up

Reply:
IC, great.